react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1137 @@
+* screens: If you have multiple screens like auth screens: login, register and profile screens, product screens it can be saved here.
+•store: If you are using Redux, then there must be action and reducers files that can be put here.
+* styles: Here we place the explore feature’s components related styles.
+* utils: All the utils/helpers files go here that storing reusable methods and logic like validations, progress bar, date pickers, and according to your app requirements.
+          </aside>
+        </section>
+
+        <section class="topic-name-slide">
+          <h4>Core Components</h4>
+        </section>
+
+        <section class="table-slide">
+          <h4 class="slide-name">Core Components</h4>
+          <table class="fragment" data-fragment-index="2">
+            <thead>
+              <tr>
+                <th>REACT NATIVE UI COMPONENT</th>
+                <th>WEB ANALOG</th>
+                <th>DESCRIPTION</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>&lt;View&gt;</td>
+                <td>A non-scrolling &lt;div&gt;</td>
+                <td>Container</td>
+              </tr>
+              <tr>
+                <td>&lt;Text&gt;</td>
+                <td>&lt;p&gt;</td>
+                <td>Displays, styles, and nests strings of text</td>
+              </tr>
+              <tr>
+                <td>&lt;Image&gt;</td>
+                <td>&lt;img&gt;</td>
+                <td>Displays different types of images</td>
+              </tr>
+              <tr>
+                <td>&lt;TextInput&gt;</td>
+                <td>&lt;input type="text"&gt;</td>
+                <td>A generic scrolling container</td>
+              </tr>
+              <tr>
+                <td>&lt;ScrollView&gt;</td>
+                <td>&lt;div&gt;</td>
+                <td>Allows the user to enter text</td>
+              </tr>
+            </tbody>
+          </table>
+          <aside class="notes">
+            Components are the building blocks of any React Native application. Components are an abstraction layer on top of the underlying native platform. On an iOS device, a React Native component is ultimately rendered as a UIView. On Android, the same component would be rendered as an android.view.
+•View:
+The most fundamental component for building a UI, View is a container that supports layout with flexbox, style, some touch handling, and accessibility controls. View is designed to be nested inside other views and can have 0 to many children of any type.
+•Text:
+A React component for displaying text. Text supports nesting, styling, and touch handling. (Nested text example). The <Text> element is unique relative to layout: everything inside is no longer using the Flexbox layout but using text layout. This means that elements inside of a <Text> are no longer rectangles, but wrap when they see the end of the line.(Containers
```

Comment:
tags `<Text> `and `<View>` aren't visible in the notes.
try escaping `<>` braces or using symbols. And please check on other slides too.

Reply:
Done